templates/generate_template.sh

```diff
@@ -49,16 +48,16 @@ samplesheet="${genScripts}/${filePrefix}.csv" ; mac2unix "${samplesheet}"
 ### Which pipeline to run
 sampleSheetColumnNames=()
 sampleSheetColumnOffsets=()
-IFS="${SAMPLESHEET_SEP}" sampleSheetColumnNames=($(head -1 "${_samplesheet}"))
+IFS="${SAMPLESHEET_SEP}" sampleSheetColumnNames=($(head -1 "${samplesheet}"))
 for (( _offset = 0 ; _offset < ${#sampleSheetColumnNames[@]:-0} ; _offset++ ))
 do
-	_sampleSheetColumnOffsets["${sampleSheetColumnNames[${_offset}]}"]="${_offset}"
+    _sampleSheetColumnOffsets["${sampleSheetColumnNames[${_offset}]}"]="${_offset}"
 done
 if [[ ! -z "${sampleSheetColumnOffsets['pipeline']+isset}" ]]; then
```

Comment:
_sampleSheetColumnOffsets?

Reply:
Stukje dat Pieter heeft geschreven. De header van de samplesheet wordt zo comma scheiden en opgeslagen in een array. de kolom die pipeline heet kan zo worden gepakt, met die pipeline wordt de rest gerund. Misschien nog even aan marloes vragen waarom als de waarde pipeline in de samplesheet leeg is er dan automatisch de diagnostics pipeline wordt gerunt.